buildconfig/Jenkins/check_for_changes

```diff
@@ -46,6 +46,26 @@ case "$TYPE" in
 	    exit $NOTFOUND
 	fi
     ;;
+    dev-docs-only)
+	# FOUND=1 iff changes are limited to dev-docs only
+	# Find all changed files and grep for required type. -v inverts match so grep=0 means
+	# there are other changes besides this
+	if git diff --name-only ${BRANCH_TIP} ${BRANCH_BASE} -- | grep -q -E -v '^dev-docs/'; then
```

Comment:
Why not
```shell
if $(! git diff --quiet ${BRANCH_TIP} ${BRANCH_BASE} -- dev-docs)
then
  exit $FOUND
fi
exit $NOTFOUND
```
Someone more thoughtful could probably make it a one-liner.

Reply:
Even shorter option (for the lines above too):
```shell
git diff --quiet ${BRANCH_TIP} ${BRANCH_BASE} -- dev-docs || return $FOUND
return $NOTFOUND
```
Then the things earlier could be single lines (for finding) that ends in `$NOTFOUND`. It's clearer to me, but maybe not others.

Reply:
Thanks for these. The script looks a little cleaner now.